tools/pipelines/templates/build-npm-package.yml

```diff
@@ -1,598 +1,540 @@
+
+                    if [[ ${{ variables['Build.Reason'] }} == "PullRequest" ]];
+                    then
+                      echo "TargetBranchName=$(targetBranchName)"
+                    fi
+
+                    # Error checking
+
+                    if [[ "$(release)" == "release" ]]; then
+                      if [[ "${{ variables.canRelease }}" == "False" ]]; then
+                        echo "##vso[task.logissue type=error]Invalid branch ${{ variables['Build.SourceBranch'] }} for release"
+                        exit -1;
+                      fi
+                      if [ -f "lerna.json" ]; then
+                        grep -e fluid.*[0-9]-[0-9] `find packages -name 'package.json'`
+                      else
+                        grep -e fluid.*[0-9]-[0-9] `find . -name 'package.json'`
+                      fi
+                      if [[ $? == 0 ]]; then
+                        echo "##vso[task.logissue type=error]Release shouldn't contain prerelease dependencies"
+                        exit -1;
+                      fi
+                    fi
+
+                    if [[ "$(release)" == "prerelease" ]]; then
+                      if [[ "${{ parameters.buildNumberInPatch }}" == "true" ]]; then
+                        echo "##vso[task.logissue type=error] Prerelease not allow for builds that put build number as the patch version"
+                        exit -1;
+                      fi
+                    fi
+
+                    if [[ "$(release)" != "prerelease" ]]; then
+                      if [[ "${{ parameters.packageTypesOverride }}" == "alpha" || "${{ parameters.packageTypesOverride }}" == "beta" ]]; then
+                        echo "##vso[task.logissue type=error]This release type is not supported. alpha/beta ***prerelease*** is allowed"
+                        exit -1;
+                      fi
+                    fi
+
+                    if [[ "$(release)" != "none" ]] && [[ "$(release)" != "" ]];
+                    then
+                      if [[ "${{ variables.publish }}" != "True" ]]; then
+                        echo "##vso[task.logissue type=error]'$(release)'' is set but package is not published. Either the branch doesn't default to publish or it is skipped."
+                        exit -1;
+                      fi
+                    fi
+              - template: /tools/pipelines/templates/include-use-node-version.yml@self
+              - template: /tools/pipelines/templates/include-install.yml@self
+                parameters:
+                  packageManager: '${{ parameters.packageManager }}'
+                  buildDirectory: '${{ parameters.buildDirectory }}'
+                  packageManagerInstallCommand: '${{ parameters.packageManagerInstallCommand }}'
+              - '${{ if eq(parameters.taskPublishBundleSizeArtifacts, false) }}':
+                  - template: >-
+                      /tools/pipelines/templates/include-set-package-version.yml@self
```

Comment:
Why would it split this into two lines? This doesn't seem that long. If we go back to relative paths it won't be an issue, but even if we stick with absolute ones, I'd put this in a single line like it was before.